orchestrator-api/src/main/java/com/sequenceiq/cloudbreak/orchestrator/OrchestratorBootstrapRunner.java

```diff
@@ -143,23 +143,28 @@ private Boolean checkResult(Boolean success, int retryCount, Exception actualExc
         }
 
         String cause = null;
+        String messageTemplate;
+        long elapsedTimeRounded = Math.max(Math.round((double) retryCount * SLEEP_TIME / MS_IN_SEC / SEC_IN_MIN), MINIMUM_DISPLAYED_TIME_IN_MIN);
         if (actualException != null) {
             cause = actualException.getMessage();
+            messageTemplate = success == null
+                    ? "Timeout: Orchestrator component failed to finish in %d min(s), last message: %s"
+                    : "Failed: Orchestrator component went failed in %d min(s), message: %s";
+        } else {
+            messageTemplate = success == null
+                    ? "Timeout: Orchestrator component failed to finish in %d min(s)"
+                    : "Failed: Orchestrator component went failed in %d min(s)";
         }
-        String messageTemplate = success == null
-                ? "Timeout: Orchestrator component failed to finish in %d min(s), last message: %s"
-                : "Failed: Orchestrator component went failed in %d min(s), message: %s";
-        long elapsedTimeRounded = Math.max(Math.round((double) retryCount * SLEEP_TIME / MS_IN_SEC / SEC_IN_MIN), MINIMUM_DISPLAYED_TIME_IN_MIN);
         String errorMessage = String.format(messageTemplate, elapsedTimeRounded, cause);
```

Comment:
What happens if the template requires 1 param when the actualException is null, but we pass 2 params with the cause to String.format?

Reply:
According to the Java Doc: `If there are more arguments than format specifiers, the extra arguments are ignored.`

Reply:
Cool.